transcom/mymove
# B 20744 int Fix error when Customer doesn't select SIT Expected

## [B-20744](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A983606)

## Summary

This fixes a bug when the Customer selects No to SIT Expected question during PPM shipment creation but then a storage type moving expense is later created. The bug is when the Service Counselor attempts to view and submit the approval for the Storage type moving expense on the Review Documents page. 

To resolve this we know that since a moving expense of Storage was submitted that SIT was indeed selected so we set the SIT Expected value to true.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. As a Customer create a new PPM shipment
2. When asked make sure you select **SIT Expected** to **No**
3. Submit the shipment
4. Login as a Service Counselor and fix the Orders section and submit the move
5. Login as the Customer again and click Upload PPM Documents
6. Fill out the first and second pages
7. On the third page under Expenses at the bottom and add a new Expense
8. Make sure the expense is STORAGE type
9. Fill out fields
10. Click Save
11. Click Save on the next page
12. Submit your documents
13. Login as the SC again, go to the PPM Closeout Queue and select your shipment
14. Click the Review Documents button
15. Click Accept on the first document and click Continue
16. Verify there aren't any "Something went wrong" error messages like in this screenshot
![image](https://github.com/user-attachments/assets/1218ecd3-2a3d-49f7-b0e8-5e20069fb7fe)
17. Make sure all fields are filled out and click Accept and Continue
18. Make sure you don't see any errors

### Frontend


### Backend
